go/pkg/pwapi/api_admin-challenge-redump.go

```diff
@@ -32,7 +32,7 @@ func (svc *service) AdminChallengeRedump(ctx context.Context, in *AdminChallenge
 	err = svc.db.
 		Model(pwdb.ChallengeInstance{}).
 		Where("flavor_id IN (?)", flavorIDs).
-		Update(&pwdb.ChallengeInstance{
+		Updates(&pwdb.ChallengeInstance{
 			Status:         pwdb.ChallengeInstance_NeedRedump,
 			InstanceConfig: []byte{},
```

Comment:
since it's a call to `Updates()`, gorm will "skip" `nil` entries, but will "set to null" empty-but-not-nil variables

so in this specific case, I confirm that we need to keep `[]byte{}`